config/routes.rb

```diff
@@ -4,12 +4,13 @@
 
     get "static_pages/help"
     get "/signup", to: "users#new"
-    resources :users
-    resources :account_activations, only: [:edit]
-
     get "/login", to: "sessions#new"
     post "/login", to: "sessions#create"
     delete "/logout", to: "sessions#destroy"
+
+    resources :users
+    resources :account_activations, only: [:edit]
```

Comment:
Khi chỉ sử dụng resource cho 1 action, em có thể viết gọn gọn lại như này nhé:
```suggestion
    resources :account_activations, only: :edit
```